components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -291,6 +292,7 @@ handler.on(PanelActions.approveHardwareTransaction.getType(), async (store: Stor
       refreshTransactionHistory(txInfo.fromAddress)
       await store.dispatch(PanelActions.setSelectedTransaction(txInfo))
       await store.dispatch(PanelActions.navigateTo('transactionDetails'))
+      apiProxy.panelHandler.setCloseOnDeactivate(true)
```

Comment:
Is this handler defined twice?

Reply:
`const apiProxy = getWalletPanelApiProxy()` is defined once
`apiProxy.panelHandler.setCloseOnDeactivate(true)` is called in for 2 separate instances 
once for `Ledger` success and once for `Trezor` success